usr/share/gamescope-session-plus/sessions.d/steam-plus

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+STEAM_SESSION_FILE=/usr/share/gamescope-session-plus/sessions.d/steam
+[[ -f "${STEAM_SESSION_FILE}" ]] && . "${STEAM_SESSION_FILE}"
+
+CURSOR_FILE="${HOME}/.local/share/Steam/tenfoot/resource/images/cursors/arrow_right.png"
+CLIENTCMD="opengamepadui --only-qam -- $CLIENTCMD"
```

Comment:
There is a  problem with this. We depend on the steam session, which is not what we want. It is very likely we will need different arguments in the future that will be incompatible with the standalone session.

I also have a question, how will users override the client command in . config if we are sourcing from the steam script?

Reply:
Depending on the steam session is exactly what we want for the steam big picture plus session. We need all the environment that is setup there for Steam. If we need to alter the arguments we can still do that.

If users override `CLIENTCMD` it will override the entire contents of the variable.

Also, it is probably better to create your own session files instead of overriding `CLIENTCMD` at this point.